src/compute/src/logging/compute.rs

```diff
@@ -289,24 +285,28 @@ impl LirMetadata {
     }
 }
 
+pub(super) struct Return {
+    pub collections: BTreeMap<LogVariant, LogCollection>,
+}
+
 /// Constructs the logging dataflow for compute logs.
 ///
 /// Params
-/// * `worker`: The Timely worker hosting the log analysis dataflow.
+/// * `scope`: The Timely scope hosting the log analysis dataflow.
+/// * `scheduler`: The timely scheduler to obtainer activators.
 /// * `config`: Logging configuration.
 /// * `event_queue`: The source to read compute log events from.
-pub(super) fn construct<A: Allocate + 'static>(
-    worker: &mut timely::worker::Worker<A>,
+pub(super) fn construct<A: Scheduler + 'static, S: Scope<Timestamp = Timestamp>>(
+    mut scope: S,
+    scheduler: A,
     config: &mz_compute_client::logging::LoggingConfig,
     event_queue: EventQueue<Column<(Duration, ComputeEvent)>>,
+    compute_event_stream: impl IntoIterator<Item = StreamCore<S, Column<(Duration, ComputeEvent)>>>,
     shared_state: Rc<RefCell<SharedLoggingState>>,
-) -> BTreeMap<LogVariant, LogCollection> {
+) -> Return {
     let logging_interval_ms = std::cmp::max(1, config.interval.as_millis());
-    let worker_id = worker.index();
-    let worker2 = worker.clone();
-    let dataflow_index = worker.next_dataflow_index();
 
-    worker.dataflow_named("Dataflow: compute logging", move |scope| {
+    scope.scoped("Dataflow: compute logging", move |scope| {
```

Comment:
I think we should remove the `Dataflow: ` prefix for the regions of the logging dataflow, to avoid confusion.